src/aggregate.ts

```diff
@@ -157,4 +196,58 @@ class Count extends AggregateField {
   }
 }
 
+/**
+ * A PropertyAggregateField is a class that contains data that defines any
+ * aggregation that is performed on a property.
+ *
+ */
+abstract class PropertyAggregateField extends AggregateField {
+  property_?: string;
+  abstract operator: string;
+
+  /**
+   * Build a PropertyAggregateField object.
+   *
+   * @param {string} property
+   */
+  constructor(property: string) {
```

Comment:
nit: 
```js
constructor(public property_: string) {
```

and remove the fields + the assignment

Reply:
done